jupyterlab/staging/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupyterlab/application-top",
-  "version": "3.2.5",
+  "version": "3.3.0-alpha.15",
```

Comment:
Here we manually update the `staging` files just to be able to realign the versions.

Normally these files should not be updated in regular PRs: https://github.com/jupyterlab/jupyterlab/issues/7237